package.json

```
@@ -32,6 +32,13 @@
   "resolutions": {
     "@storybook/react-docgen-typescript-plugin": "1.0.6--canary.9.cd77847.0"
   },
+  "scripts": {
+    "prepare": "husky"
+  },
+  "lint-staged": {
+    "**/*.css": "yarn stylelint --fix",
```

Nit: Can we change this to just `*.css` instead of `**/*.css`? Since we're doing `*.{js,ts,tsx,json,md,yml}` for the other one, not `**/*.{js,ts,tsx,json,md,yml}`.


```suggestion
    "*.css": "yarn stylelint --fix",
```